packages/next/src/root-layout/index.tsx

```diff
@@ -39,23 +39,7 @@ export async function RootLayout( {
  * @return Merged metadata.
  */
 export async function generateRootMetaData(): Promise< Metadata > {
```

Comment:
To keep things stackable and pluggable, we should support/pass through the `params` and `searchParams` (ref https://nextjs.org/docs/app/api-reference/functions/generate-metadata#generatemetadata-function)